src/components/NewOrganizationModal/NewOrganizationModal.js

```diff
@@ -17,21 +16,25 @@ const ModalInputRow = styled(InputRow)`
   display: block;
   border: none;
 `
-const NewOrganizationModal = ({ isOpen, onDismiss, onSubmit }) => {
-  const formik = useFormik({
```

Comment:
Formik was overkill for this single input form. Also it caused an infinite rerender when using it with the useEffect below so I scrapped it.